submissions/titilayoOlutola/HTML/Html-task.html

```diff
@@ -21,37 +21,15 @@ <h4> <em>your-library-on-the-go</em></h4>
         <ul>
             <li><a href="#">Home</a></li>
             <li><a href="#">Search</a></li>
-            <li><a href="#">Contact Us</a></li>
+            <li><a href="C:\Users\laptop\Desktop\Dufuna-Fem\submissions\titilayoOlutola\html-forms\contact-form.html">Contact Us</a></li>
```

Comment:
This file path is absolute and will only work at your end. It wont work for me because I don't have this directory on my computer. You can simply use '../html-forms/contact-form.html'

Also, you should include the link to the product page i.e Html-task.html as well.

The two links should be on both pages you have worked on just as stated in the task criteria.

Reply:
Changing the highlighted file path makes my contact form inaccessible. when I changed it to '../html-forms/contact-form.html', the response I got was that my contact form can't be accessed.